app/scripts/webcam.js

```diff
@@ -4,7 +4,7 @@
  * (c) Jonas Hartmann http://jonashartmann.github.io/webcam-directive
  * License: MIT
  *
- * @version: 3.2.1
+ * @version: 3.2.2
```

Comment:
We might just want to remove this line to avoid confusion since the only place that really matters is the package.json file anyways

```suggestion
 * @version: 3.3.1
```

Reply:
I forgot I had mentioned updating the README file to remove some outdated info. Can we update it to remove the following 3 things:

note: maybe for the installation example we just update the path to the script instead of removing it
![image](https://user-images.githubusercontent.com/1569157/174245884-3859cece-7cc4-4104-a333-20245fc9eb88.png)